.github/workflows/test.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: tests
+on: [ push, pull_request ]
+jobs:
+  ruby:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ '2.3', '2.4', '2.5' ]
```

Comment:
Any reason not to run it against newer versions of ruby?

Reply:
I'm just coping the CI build as-is for the moment. We can look at upgrading things in future PRs.